drivers/clock_control/Kconfig.nrf

```diff
@@ -67,3 +74,5 @@ config CLOCK_CONTROL_NRF_K32SRC_20PPM
 endchoice
 
 endif # CLOCK_CONTROL_NRF
+
+endif #!CLOCK_CONTROL_OUT_OF_TREE_DRIVER
```

Comment:
Somebody is going to come with this nit, so it can be me already and save you time: Add an EOL at EOF :)

Reply:
Hah, thanks. I'll do it right away.

Reply:
Fixed for all three `Kconfig.nrf.*` files